include/zephyr/dfu/mcuboot.h

```diff
@@ -248,6 +247,22 @@ int boot_request_upgrade_multi(int image_index, int permanent);
  */
 int boot_erase_img_bank(uint8_t area_id);
 
+/**
+ * @brief Get the offset of the status in the image bank
+ *
+ * @param area_id flash_area ID of image bank to get the status offset
+ * @return a positive offset on success, negative errno code on fail
+ */
+ssize_t boot_get_area_trailer_status_offset(uint8_t area_id);
+
+/**
+ * @brief Get the offset of the status from an image bank size
+ *
+ * @param area_size size of image bank
+ * @return offset of the status
```

Comment:
Add note that negative value means that the trailer will not fit into given size and user needs to take care so that data of an image will not cross the offset.